extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/NamedRunner.java

```diff
@@ -27,7 +27,6 @@
  * marked with @After and @Before (but also the Test method itself) can request
  * this name for which it is running.
  *
- * @author Rudy De Busscher
  */
 public class NamedRunner extends BlockJUnit4ClassRunner {
```

Comment:
This code appears to be copied from Apache myfaces project.  We should preserve the author tag and possibly add to NOTICE.